### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#362] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152793522-29b3b789-4e51-4cc7-9516-d4b6a58e6383.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/211193098-a0d7bffa-ea41-47e4-bf5a-b6bf604f0f51.png)
